PacketDefinitions420/PacketNotifier.cs

```diff
@@ -721,23 +721,71 @@ public void NotifyLeaveVision(IGameObject o, TeamId team)
             }
         }
 
-        public void NotifyEnterVision(IGameObject o, TeamId team)
+        // TODO: Fix IsAnythingBetween randomly losing vision often, which ends up spamming this as well
+        public void NotifyEnterVisibilityClient(IAttackableUnit u, TeamId team, int userId = 0)
         {
-            switch (o)
+            var enterVis = new OnEnterVisiblityClient(); // TYPO >:(
+            var itemData = new List<ItemData>(); //TODO: Fix item system so this can be finished
+            enterVis.Items = itemData;
+            var shields = new ShieldValues(); //TODO: Implement shields so this can be finished
+            enterVis.ShieldValues = shields;
+            var charStackDataList = new List<CharacterStackData>();
+            var charStackData = new CharacterStackData
+            {
+                SkinName = u.Model,
+                OverrideSpells = false,
+                ModelOnly = false,
+                ReplaceCharacterPackage = false,
+                ID = 0
+            };
+            enterVis.LookAtPosition = new Vector3(1, 0, 0);
+            if (u is IObjAiBase)
+            {
+                _ = ((IObjAiBase)u).GetBuffs().ToList(); // buffList left for when someone decides to finish this
```

Comment:
This is major style issue. Don't use `_` for naming. Why do you need the casting? I think it's redundant.